pkg/skaffold/docker/image.go

```diff
@@ -146,8 +147,23 @@ func (l *localDaemon) Close() error {
 	return l.apiClient.Close()
 }
 
-func (l *localDaemon) ContainerLogs(ctx context.Context, out io.Writer, id string, muter chan bool) (io.ReadCloser, error) {
-	return l.apiClient.ContainerLogs(ctx, id, types.ContainerLogsOptions{ShowStdout: true, ShowStderr: true, Follow: true})
+// ContainerLogs streams logs line by line from a container in the local daemon to the provided PipeWriter.
+func (l *localDaemon) ContainerLogs(ctx context.Context, w *io.PipeWriter, id string) error {
+	r, err := l.apiClient.ContainerLogs(ctx, id, types.ContainerLogsOptions{ShowStdout: true, ShowStderr: true, Follow: true})
+	if err != nil {
+		return err
+	}
+	rd := bufio.NewReader(r)
+	for {
+		s, err := rd.ReadString('\n')
```

Comment:
hmm that's a really good point. I honestly don't know but I'm guessing this would cause the `\r` to show up in the logs. do you think it would be enough to check if the host platform is Windows? my understanding is that you can only run Windows containers on Windows Docker itself, so if I add a check for `runtime.GOOS == Windows` and then use `\r\n` as the delimiter I think it might work.